### ISSUE TITLE
Contao 5 Support

### ISSUE THREAD

**Issue by tinokramm** [alnv/catalog-manager#370]
Hallo. Ist bereits absehbar, wann der Catalog Manager kompatibel zu Contao 5 wird?

**Comment by hoaxone** [comment]
Hi, gibt es hierzu schon Neuigkeiten bzw. eine Schätzung?

**Comment by Shoekrates** [comment]
Hallo Alexander, die Anfragen oben sind jetzt schon einige Zeit her. Gibt es mittlerweile eine zeitliche Planung für die Contao 5-Kompatibilität?

**Comment by jankout** [comment]
Hallo Alexander, Ich würde mich auch darüber sehr freuen, das bezahlte Produkt auch für 5.3.x nutzen zu können.

**Comment by tinokramm** [comment]
Hej Alexander, tausend Dank für Deine Mühe und diese wirklich wichtige Erweiterung im Contao Kosmos. Es wäre jetzt langsam wirklich wichtig zu wissen, ob bzw. wann der Contao 5 Support kommt.

Wir möchten Deine Erweiterung bei einem kommenden Projekt gern wieder einsetzen. Noch basiert die Website auf Contao 4.13. Ungern möchten wir da viel Zeit und Aufwand hineinstecken, wenn ein Update auf Contao 5.3 womöglich gar nicht funktionieren wird. Dazu wäre Feedback bitte sehr sehr wichtig, damit wir Agenturen Planungssicherheit bekommen.

Danke schon vorab.

Beste Grüße

Tino

**Comment by alnv** [comment]
Ich mache die Tage eine offizielle Ankündigung auf catalog-manager.org, wollte erstmal abwarten, bis C5.3 kommt.

Es wird eine neue Catalog-Manager-Version (CM 3) speziell für Contao 4.13 und 5 geben: https://github.com/alnv/contao-catalog-manager-bundle … CM 3 ist auf gutem Weg und wird bald für Contao5 zur Verfügung gestellt. Features wie FE-Editing/CSV-Import etc. werden nach und nach nachgezogen.

CM 3/ CM 2 sind aber nicht rückwärts kompatibel zu CM 1, das ist dem geschuldet, dass wir damals CM 1 für Contao 3.5 gemacht haben.

**Comment by tinokramm** [comment]
Hej Alexander, das hört sich doch schon mal gut. Sollte man bei Websites mit Contao 4.13 auf CM3 warten? Oder ist es unproblematisch mit CM2 zu starten und dann später auf CM3 und Contao 5.3 zu updaten?

**Comment by alnv** [comment]
Ich empfehle mit CM 2 und Contao 4.13 zu starten und dann auf Contao 5 updaten.

**Comment by andre5tz** [comment]
Hallo Alexander,

bleibt es denn dabei, dass CM1 und CM2/CM3 NICHT gemeinsam in einer Contao-Installation installiert werden können?
Aktuell gibt es den Hinweis, weil ansonsten mit einer kaputten Datenbank zu rechnen ist: 
`Wichtig: Diese Erweiterung nicht mit Catalog-Manager v1 installieren!`
Würde mich freuen, wenn Du hierzu dann auch eine klare Aussage treffen könntest.
Vielen Dank.

VG
André

**Comment by alnv** [comment]
Ja, das bleibt dabei … wer CM 1 auf CM 2/3 aktualisiert, der muss vorab CM 1 deinstallieren (auch Tabellen in der DB löschen) und dann CM 2/3 installieren.

**Comment by mapfei** [comment]
Hallo Alexander! Muss bei der Verwendung von CM1 und einem Update auf Contao 5.3 und CM3 alles neu aufgebaut werden oder gibt es eine Möglichkeit der Migration?
